app/src/main/java/com/kickstarter/ui/viewholders/CreatorDashboardReferrerBreakdownViewHolder.java

```diff
@@ -147,4 +149,9 @@ private void setAveragePledgeTextViewText(final @NonNull Pair<Project, Integer>
     final String amountString = this.ksCurrency.format(projectAndAveragePledge.second, projectAndAveragePledge.first, true, true, RoundingMode.DOWN);
     this.averagePledgeAmountTextView.setText(amountString);
   }
+
+  private void setDrawableColor(final @NonNull TextView view, final @NonNull Integer color) {
```

Comment:
Since we'll be switching to an `ImageView`, we can change the color by doing `imageView.setColorFilter(color)`

Reply:
oooo yes good point ty ty

Reply:
and we can go back to point free 👯